modules/zivid/__init__.py

```diff
@@ -24,3 +24,8 @@
 from zivid.settings import (
     Settings,  # noqa: F401 'zivid.settings.Settings' imported but unused
 )
+from zivid.settings_2d import (
+    Settings2d,  # noqa: F401 'zivid.settings.Settings2d' imported but unused
```

Comment:
Small d? Intentional?

Reply:
It was intentional, but I do not have strong opinions about this, can change it to Settings2D.